autopush/web/webpush.py

```diff
@@ -80,9 +84,43 @@ def validate_uaid(self, d):
                                  status_code=410,
                                  errno=105)
 
+        if result["router_type"] == "webpush":
+            self._validate_webpush(d, result)
+
         # Propagate the looked up user data back out
         d["user_data"] = result
 
+    def _validate_webpush(self, d, result):
+        settings = self.context["settings"]  # type: AutopushSettings
+        log = self.context["log"]  # type: Logger
```

Comment:
Agreed, though it is a validate's log messages so that seems ok. We don't use the origin log name anyways.